CMakeLists.txt

```diff
@@ -151,15 +151,12 @@ if(ENABLE_BUILD_VMM)
         DEPENDS bfunwind
     )
 
-    vmm_extension(
-        bfvmm_main
+    add_subproject(
+        bfvmm_main vmm
         DEPENDS bfvmm
-        DEPENDS bfintrinsics
-        SOURCE_DIR ${SOURCE_EXAMPLE_DIR}/bfvmm_main/src
```

Comment:
Are we planning to put the example directory back?

Reply:
Absolutely. All of the existing examples will be ported to this new design and included in the base repo. Once this is done, all of the example repos will be deprecated.